src/openrct2/platform/posix.c

```diff
@@ -844,6 +845,36 @@ utf8* platform_get_username() {
     }
 }
 
+bool platform_get_steam_path(utf8 *outPath, size_t outSize)
```

Comment:
The whole point of posix.c is to include stuff common to macOS and Linux/BSD. Since this is not common, please add this method to the respective files, rather than posix.c.